src/utils/trust/details.mapper.ts

```diff
@@ -9,8 +9,9 @@ import { addTrustToBeneficialOwner, getTrustByIdFromApp, removeTrustFromBenefici
 const mapDetailToPage = (
   appData: ApplicationData,
   trustId: string,
+  isReview?: boolean
```

Comment:
could drop the "?" as probably always set or could be set?

Reply:
Done.